infra/email.js

```diff
@@ -20,11 +21,14 @@ if (!webserver.isServerlessRuntime) {
 
 const transporter = nodemailer.createTransport(transporterConfiguration);
 
-async function send({ from, to, subject, text }) {
+async function send({ from, to, subject, component }) {
+  const { html, text } = render(component);
```

Comment:
Será que faz sentido o `infra/email` depender do `@tabnews/mail`?

Não é melhor ele receber o `html` pronto de quem for enviar o email:

```suggestion
async function send({ from, to, subject, html, text}) {
```